backend/src/layer/data/swka_parser/swka_link_creator.rs

```diff
@@ -132,4 +134,12 @@ mod tests {
         let result = test_util::get_creator().get_all_urls_for_next_weeks_from_date(date);
         assert_eq!(result, URLS_FOR_NEXT_WEEKS);
     }
+
+    #[test]
+    fn test_get_todays_date() {
```

Comment:
i dnt think this is a helpful test (testing a function against its implementation). If you want to get coverage for this function, there should be a test which tests the function where this is called

Reply:
It is for sure not. The problem is, we can only test this function with date related tests. So test only will work for one day. There's no "easy" way to create a test case that calls this function. We can remove it, as it is very dumb, but good for coverage :)

Reply:
is there no function calling `get_all_urls`?
if you only call this function once (even if not) you could imo just inline it.